src/sway.h

```diff
@@ -8,13 +8,13 @@
  * @param[in] window identifier of currently focused window (container)
  * @return keyboard layout to set, -1 to leave current
  */
-typedef int (*on_focus)(int window);
+typedef int (*on_focus)(const char* window_key);
```

Comment:
`window_key` that was created by the function `get_cache_key()`.
I think we should use one one name for this: either `const char* cache_key` or `get_window_key()`.

Reply:
Renamed to get_window_key